pom.xml

```diff
@@ -170,17 +170,20 @@
         <xalan.bundle.version>2.7.2_3</xalan.bundle.version>
         <!--END versions that MUST exactly match the versions used by the CXF and Camel features-->
         <!-- Artemis dependencies -->
-        <artemis.version>2.1.0</artemis.version>
+        <artemis.version>2.2.0</artemis.version>
         <!--START Camel and Artemis features should use the same versions for these bundles-->
-        <proton.version>0.19.0</proton.version>
-        <qpid-jms.version>0.23.0</qpid-jms.version>
+        <proton.version>0.20.0</proton.version>
+        <qpid-jms.version>0.24.0</qpid-jms.version>
         <!--END versions that should be kept in sync between Camel and Artemis features-->
         <jboss-logging.version>3.3.0.Final</jboss-logging.version>
         <jgroups.version>3.6.13.Final</jgroups.version>
         <commons-beanutils.version>1.9.3</commons-beanutils.version>
         <netty.version>4.1.9.Final</netty.version>
+        <commons-beanutils.version>1.9.2</commons-beanutils.version>
+        <netty.version>4.1.14.Final</netty.version>
```

Comment:
Update the existing `netty.version` instead.

Reply:
And beanutils

Reply:
oh thanks, I think I resolved the conflicts wrong.

Reply:
Fixed